### DISCUSSION TITLE
postmarketOS: Phosh

### DISCUSSION THREAD

**Discussion by hansemro** [hansemro/kc1-linux#35] [Show and tell]
![phosh](https://user-images.githubusercontent.com/40348686/117054676-36a62d00-accf-11eb-800d-81b1be0138a0.gif)

Very slow  :snail:

Interesting note: lockscreen password is limited to numbers !? I had to change my password just so that I can login.

I am probably missing a step somewhere, but this what I basically did

```
apk add postmarketos-ui-phosh
# make sure to exit out of all X programs
# run phosh from a virtual terminal ttyN
openvt -f -c N phosh
```

To quit:
```
killall phosh
killall phoc
```

Additional note: weston can work, but it is also very slow.

**Comment by mj-sakellaropoulos** [comment]
> very slow

Yeah, hardware acceleration is not really working - I tried various packages just for fun, they either don't work or have errors

it was something like:
```
xf86-video-omap
pvr-mesa-classic
```

**Comment by Newbytee** [comment]
> Interesting note: lockscreen password is limited to numbers !? I had to change my password just so that I can login.

This is no longer the case btw. You can now use passwords with any letter Squeekboard lets you type.
